code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -1,600 +1,600 @@
-/obj/item/robot_module

```

Comment:
Putting my foot down on these PRs that overwrite the entire file; it's making git blame and history very difficult to navigate.

You'll have to resolve this or resubmit in a version that doesn't have this; I figured github would handled this more gracefully, on merge...but it didn't; files that get overwritten like this show up in the git blame+history; not good at all.